### ISSUE TITLE
birdview pictures are all black, is it normal?

### ISSUE THREAD

**Issue by beyondpzk** [wayveai/mile#24]
![image](https://github.com/wayveai/mile/assets/35320670/ad5afc32-e824-4653-b3df-bc4046667e84)

**Comment by anthonyhu** [comment]
Yes because they are saved as unique ID from [0, n]. To visualise them with colours you need to convert those ID to RGB values using https://github.com/wayveai/mile/blob/main/mile/constants.py#L24

**Comment by Syk-yr** [comment]
But the maximum value in the bird view is more than 8, but there are only 8 colors. What are the extra colors?
